src/main/resources/ath-container/Dockerfile

```diff
@@ -33,14 +33,15 @@ RUN apt-get update && \
 
 # Install a fixed firefox version that is known to work with the current selenium version, copied from https://hub.docker.com/r/selenium/node-firefox/dockerfile
 ARG FIREFOX_VERSION=108.0.2
+# hadolint ignore=DL3008, DL3015
```

Comment:
```suggestion
# hadolint ignore=DL3008
```

DL3015 should not be ignored: it relies on a set of implicit additional dependencies (the "recommended" packages) that can change over time even on a fixed disribution.

Better to add the `--no-install-recommends` and add the required packages (as these packages... are needed).
Example is `ca-certificates` which is recommended by `curl` but should be mande explicit most of the time.